db/migrate/20170626122608_set_public_body_translation_defaults.rb

```diff
@@ -0,0 +1,20 @@
+# -*- encoding : utf-8 -*-
+class SetPublicBodyTranslationDefaults < ActiveRecord::Migration
+  def up
+    change_column :public_body_translations,
+                  :short_name, :text, :null => false, :default => ""
+    change_column :public_body_translations,
+                  :notes, :text, :null => false, :default => ""
+    change_column :public_body_translations,
+                  :publication_scheme, :text, :null => false, :default => ""
+  end
```

Comment:
\*sigh\*

```rb
PublicBody::Translation.where(name: nil).map(&:locale).uniq
=> [:cy]
PublicBody::Translation.where(short_name: nil).map(&:locale).uniq
=> [:cy]
```

Maybe we also need to check what creating a new PublicBody via the admin interface is doing as these don't look like old data.

```rb
PublicBody::Translation.where(name: nil).last.created_at
=> Wed, 28 Jun 2017 13:21:48 BST +01:00
```